.gitattributes

```diff
@@ -31,7 +31,7 @@
 
 # configure is a shell-script; the linguist-generated attribute suppresses
 # changes being displayed by default in pull requests.
-/configure text eol=lf -diff linguist-generated
+/configure text eol=lf -diff linguist-generated diff
```

Comment:
Instead of adding `diff` shouldn't we remove `-diff` ?

Reply:
oh yes I thought it was a command-line parameter -- too much shell for today?